frontend/src/v5/store/containers/containers.redux.ts

```diff
@@ -39,8 +38,23 @@ export const INITIAL_STATE: IContainersState = {
 	filterQuery: '',
 };
 
-export const setFilterQuery = (state = INITIAL_STATE, { query }) => ({ ...state, filterQuery: query });
+export const setFilterQuery = (state = INITIAL_STATE, { query }: IContainersActions['setFilterQuery']) => (
+	{ ...state, filterQuery: query }
+);
+
+export const toggleFavourite = (state = INITIAL_STATE, { containerId }: IContainersActions['addFavourite']) => {
```

Comment:
When you change `toggleFavouriteSuccess` to `setFavouriteSuccess` remember to rename this one.